src/Symfony/Bridge/Twig/NodeVisitor/Scope.php

```diff
@@ -52,7 +52,7 @@ public function enter()
     /**
      * Closes current scope and returns parent one.
      *
-     * @return Scope|null
+     * @return self|null
```

Comment:
Avoid duplicating terms. But you're right, if not done consistently it's not really worth it anyway.